examples/authTokens/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const corbado = require('@corbado/node-sdk');
+
+const app = express();
+
+const config = new corbado.Config(process.env.CORBADO_PROJECT_ID, process.env.CORBADO_API_SECRET);
+const sdk = new corbado.SDK(config);
+
+app.get('/', async (req, res) => {
+  try {
+    const corbadoAuthToken = req.query.corbadoAuthToken;
+    const request = {
+      token: corbadoAuthToken,
+      clientInfo: {
```

Comment:
Müssen wir die clientInfo aktuell immer mitgeben? Kann mir vorstellen, dass das anstrengend/verwirrend sein könnte für devs.

Reply:
as discussed right now its mandatory, can fix it later